pkg/build/api/types.go

```diff
@@ -86,6 +95,10 @@ type BuildStatus struct {
 	// Cancelled describes if a cancel event was triggered for the build.
 	Cancelled bool
 
+	// Skipped indicates that this build was marked as skipped. The build will be
```

Comment:
@smarterclayton @bparees i'm going to remove this for now and we can revisit "skipped" builds later when we implement the auto-skip logic. Right now, users will have to cancel builds in the queue manually using `oc cancel-build`. We discussed with @bparees on IRC that it would be nice to provide a 'range' option to `cancel-build` so you don't have to list all builds... Other option is to add something like: `oc cancel-build BCNAME --all-queued` which will cancel all builds that are not running. Basically cleanup the queue. WDYT?